Altis_Life.Altis/stringtable.xml

```diff
@@ -1307,7 +1307,7 @@
             <Spanish>You must enter a message to send!</Spanish>
             <Russian>You must enter a message to send!</Russian>
             <German>Du musst eine Nachricht eingeben!</German>
-            <French>Vous devez écrire un message à envoyer !</French>
+            <French>Vous devez écrire un message à envoyer!</French>
```

Comment:
Why not leaving the space here ? Usually, there's one before it.

Reply:
True.